src/posts/blog/7-budget/budget.ts

```diff
@@ -0,0 +1,91 @@
+import podOverviewChart from './img/pod-overview-chart.png'
+import { ImageMeta } from 'posts/post'
+
+const image: ImageMeta = {
+  url: podOverviewChart,
+  width: 1632,
+  height: 502,
+}
+
+const title = 'microCOVID Risk Tracker: How our house got our lives back'
+
+const author = 'Jeremy Blanchard'
+const date = 'February 11, 2021'
+
+const summary =
+  'Navigating house/pod agreements around COVID can be really stressful. We made a tool to help make it a lot easier.'
+const content = `
+
+<style>
+img {
+  max-width: 100%;
+  height: auto;
+}
+</style>
+
+*Today we’re launching a new version of the microCOVID Risk Tracker with features for households and pods to collectively budget their risk over time.* [*You can download the latest version of the Risk Tracker here →*](/tracker)
+
+
+<figure>
+  <img src="${podOverviewChart}" alt="Graph of 4 housemate's risk budget, showing risk points used and risk points remaining." />
+</figure>
+
+
+## How the Risk Tracker helped our house get our lives back
+
+We're launching Version 2 of the [microCOVID Risk Tracker](/tracker) today, and I want to share a bit of the backstory. Up until [microCOVID](/) was released in August 2020, my house was struggling with the same thing I know many communal houses were struggling with: we want to make sure to keep ourselves, each other, and the wider community safe from COVID, *and* at the same time, there are things that are important enough that we want to be able to find a way to do them. 
+
+The dilemma: how do you make decisions about what a “safe amount of risk is” when living with many other people?
+
+At that time our system was similar to many other group houses: masked, outdoor, distance hangs were okay, but anything other than that needed group input/consensus. Having every single person’s activities be subject to a consensus discussion in a house meeting is a recipe for a ton of stress and struggle.
+
+We encountered questions like:
+* “What’s riskier: a 3-hour unmasked. outdoor hang? Or a 30-minute indoor, masked hang? 🤷‍♂️”
+* “Sam really wants to see their bodyworker. I really wanted to see my partner. But was I ok being in indirect contact with that bodyworker for Sam's sake? Did Sam think my happiness from seeing my partner was worth the risk they'd experience?”
+
+Then we disocvered the microCOVID spreadsheet. I realized it had the potential to completely change how we had these discussions. If we could quantify and track how much risk each of us were taking, we could allocate everyone a small but even share of risk that they could use to make their own decisions, without consulting the house every time.
```

Comment:
I think you want to use the phrase "risk budget" here to define it.

Perhaps add: "We agreed that each of us was willing to take on a 1% chance of getting COVID over the next year. Dividing risk evenly among each of us meant we could each bring XX microCOVIDs each week; this became our risk "budget" that we could plan against without consulting the house every time"

Reply:
Did a slight combo of the two. I think it helps ease the reader in more.